front-end/src/messages.js

```diff
@@ -0,0 +1,36 @@
+export default {
+  en: {
+    'login_login': 'Login',
```

Comment:
One of the common approaches that makes translation convenient is to use the English string as the key, e.g.:
```
en: {
'Login': 'Login",
'Password': 'Password'
}
```

```
ru: {
   'Login': 'Логин',
   'Password': 'Пароль'
}
```
In this case you would not need to waste time to come up with these ID's and it would be easier to read in the code.

Reply:
I might be wrong on this one. Reading the docs, trying to figure out how i18n works in this library.